config.yml

```diff
@@ -77,100 +73,102 @@ steps:
           - type: findInTree
             path: my-workflow.yml
             tree: "%actions.isInTree.data.tree%"
-            # multiple: true
             action_id: fileLocation
           # help user with proper file location
           - type: respond
             with: e-wrong-file-location.md
             data:
               haveFile: "%actions.fileLocation.path%"
-              needFile: ".github/actions/cat-facts/action.yml"
+              needFile: ".github/workflows/my-workflow.yml"
               editLink: "%payload.repository.html_url%/edit/%payload.pull_request.head.ref%/%actions.fileLocation.path%"
               fileName: my-workflow.yml
         # End filepath verification
         #######################################################
       - type: respond
-        issue: Start here!
+        issue: "%registration.store.welcomeIssueNumber%"
         with: new-pull-request.md
         data:
-          pullUrl: "%payload.pull_request.html_url%"
+          pullUrl: "%registration.store.firstPullUrl%"
       - type: closeIssue
-        issue: Start here!
+        issue: "%registration.store.welcomeIssueNumber%"
       - type: createReview
         body: 01_merge-workflow.md
         event: APPROVE
+        # store:
+        #   firstPullUrl: "%result.data.html_url%"
+        #   firstPullNumber: "%result.data.number%"
         data:
-          actionsUrl: "%payload.repository.html_url%/actions"
+          actionsUrl: "%payload.repository.html_url%/actions%"
 
-  - title: Merge the new workflow into the master branch
-    description: Make the workflow available for use on the repository
+  - title: Merge your first pull request
+    description: Make the workflow available to the repository by merging the pull request
     event: pull_request.closed
-    link: "{{ repoUrl }}/pull/2"
+    link: "{{store.firstPullUrl}}"
     actions:
       - type: createIssue
         title: Create an issue comment with GitHub Script
         body: 02_workflow-triggered.md
-        action_id: triggerIssue
+        store:
+          secondIssueNumber: "%result.data.number%"
+          secondIssueUrl: "%result.data.html_url%"
       - type: respond
         with: new-issue.md
-        issue: Create my-workflow.yml
+        issue: "%registration.store.firstPullNumber%"
         data:
-          issueURL: "%actions.triggerIssue.data.html_url%"
+          issueURL: "%registration.store.secondIssueUrl%"
 
   - title: Add another GitHub Script action
     description: Use GitHub Script to place this issue into a project board
-    event: check_suite.completed
-    link: "{{repoUrl}}/issues/3"
+    # event: check_suite.completed # issue_comment.created/edited
+    event: issue_comment.created
+    link: "{{store.secondIssueUrl}}"
     actions:
       - type: gate
-        left: "%payload.check_suite.app.name%"
+        left: "%payload.comment.user.login%"
         operator: ===
-        right: "GitHub Actions"
+        right: "github-actions[bot]"
```

Comment:
I think it's more meaningful for the learner to trigger the `issue_comment`, what do you think @mattdavis0351?

Reply:
I'm not a fan of it.  The course isn't "can the learning open an issue" if that makes sense.  Its focus is on GitHub Script and this is not much different than us opening Pulls for the learner when we need to.